PackageInfo.g

```diff
@@ -29,10 +29,10 @@ Persons := [
 
 SourceRepository := rec(
     Type := "git",
-    URL := "https://github.com/gap-packages/QuickCheck",
+    URL := "https://github.com/ChrisJefferson/QuickCheck",
 ),
 IssueTrackerURL := Concatenation( ~.SourceRepository.URL, "/issues" ),
-PackageWWWHome  := "https://gap-packages.github.io/QuickCheck/",
+PackageWWWHome  := ~.SourceRepository.URL,
```

Comment:
The danger with this is that one might forget to revert this change when trying to make the first actual release...

How about instead simply making a release of this package? It is being used after all, and costs nothing, so what's the harm?